_posts/2020-10-03-hacktoberfest.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  "Hacktoberfest: let's start to contribute to Strimzi"
+date: 2020-10-03
```

Comment:
Shouldn't we publish it tomorrow on _1st_ given how it ties to October?

Reply:
I agree, I was not sure about the time needed to review :-) I will change it.